packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -63,13 +78,20 @@ export default class TooltipAnchor extends React.Component<Props, State>
     _unsubscribeFromTracker: ?() => void;
     _timeoutID: ?TimeoutID;
 
+    static ariaContentId = "aria-content";
     static defaultProps = {
         forceAnchorFocusivity: true,
     };
 
-    state = {
-        active: false,
-    };
+    constructor(props: Props) {
+        super(props);
+
+        this._focused = false;
```

Comment:
Why did you move these to be inside of a constructor instead of leaving them as a global like before?

Reply:
I don't like where our linter requests class properties be initialized which is after the lifecycle methods.